src/api/goals.js

```diff
@@ -0,0 +1,37 @@
+import express from 'express';
+
+export default (dispatch, action, storage) => {
+  const router = new express.Router();
+
+  router.get('/add/:goal', (request, response) => {
+    const timer = storage.read()[request.timerId];
+
+    if (timer.goals.length >= 5) {
```

Comment:
The timer should be able to set this limit rather than a hard limit in the api, which would also let the clients make better decisions rather than guess/hard-code-sync.

Reply:
See #44